UNRELEASED.yaml

```diff
@@ -1,5 +1,9 @@
 # UNRELEASED:
 
+ADDED:
```

Comment:
This is just changing the visual appearance of an existing thing, right?
Maybe it should be `fixed` instead of `added`?

Reply:
Good point, I guess if we are removing the old assets from svgs it could be breaking right of someone is directly using them, outside of their recommended usage?

Reply:
I think it's breaking for `bpk-svgs`, patch for `bpk-mixins` and `bpk-component-radio`.